clouddq/main.py

```diff
@@ -438,6 +439,20 @@ def main(  # noqa: C901
         )
         if job_status == JobStatus.SUCCESS:
             logger.info("Job completed successfully.")
+
+            if not dry_run and target_bigquery_summary_table:
+                invocation_id = get_dbt_invocation_id(dbt_path)
```

Comment:
I suggest moving get_dbt_invocation_id() into the runners.dbt. path somehow, Maybe part of DbtRunner